Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
name = urllib.request.urlopen('https://randomincategory.toolforge.org/?category=All%20Wikipedia%20level-5%20vital%20articles&server=en.wikipedia.org&cmnamespace=&cmtype=&returntype=subject').read(500).decode().split('title>')[1].split(' - Wikipedia')[0]
content = wikipedia.page(name).content
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [1][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m name [38;5;241m=[39m [43murllib[49m[38;5;241m.[39mrequest[38;5;241m.[39murlopen([38;5;124m'[39m[38;5;124mhttps://randomincategory.toolforge.org/?category=All[39m[38;5;124m%[39m[38;5;124m20Wikipedia[39m[38;5;132;01m%20le[39;00m[38;5;124mvel-5[39m[38;5;124m%[39m[38;5;124m20vital[39m[38;5;132;01m%20a[39;00m[38;5;124mrticles&server=en.wikipedia.org&cmnamespace=&cmtype=&returntype=subject[39m[38;5;124m'[39m)[38;5;241m.[39mread([38;5;241m500[39m)[38;5;241m.[39mdecode()[38;5;241m.[39msplit([38;5;124m'[39m[38;5;124mtitle>[39m[38;5;124m'[39m)[[38;5;241m1[39m][38;5;241m.[39msplit([38;5;124m'[39m[38;5;124m - Wikipedia[39m[38;5;124m'[39m)[[38;5;241m0[39m]
[1;32m      2[0m content [38;5;241m=[39m wikipedia[38;5;241m.[39mpage(name)[38;5;241m.[39mcontent

[0;31mNameError[0m: name 'urllib' is not defined
NameError: name 'urllib' is not defined

